### PR TITLE
Add multipass info

### PR DESCRIPTION
**Pull request by pozsgaic**
Signed-off-by: pozsgaic <chris.pozsgai@progress.com>

Adds multipass specific info and separate into VMWare and Multipass sections

### CODE REVIEW

File: DEVELOPING.md [hunk 1]
```diff
@@ -8,13 +8,48 @@ There are potentially multiple ways of creating a Builder dev environment - but
...
+to /etc/pf.anchors/com.apple to enable port forwarding for builder:
+
+_rdr pass on lo0 inet proto tcp from any to self port 3000 -> 127.0.0.1 port 9636_
+
+_rdr pass on en0 inet proto tcp from any to self port 9636 -> <YOUR VM IP ADDRESS> 9636_
+
+If using wireless connection, the network value in the second line is likely en0.   
+Check by running ifconfig at the terminal and update this value if necessary. 
+
+Multipass assigns a static IP address for you when you create a virtual machine.  It can be found by running `multipass list`
```
Comment (sajjaphani) [inline DEVELOPING.md:40]: `multipass list` shows IP only for running instances

File: DEVELOPING.md [hunk 2]
```diff
@@ -8,13 +8,48 @@ There are potentially multiple ways of creating a Builder dev environment - but
 
 ## Prerequisites
 
+This document currently only contains information for running on MacOS with Linux-based VMs.  Information for running
+on Windows or other platforms is not provided in this document.  
+
+_MacOS_
```
Comment (sajjaphani) [inline DEVELOPING.md:14]: This section title may not be required.

File: DEVELOPING.md (hunk 1 above)
Comment (pozsgaic) [reply DEVELOPING.md:40]: Will mention that we should start the instance first

File: DEVELOPING.md [hunk 3]
```diff
@@ -8,13 +8,48 @@ There are potentially multiple ways of creating a Builder dev environment - but
...
+
+If using wireless connection, the network value in the second line is likely en0.   
+Check by running ifconfig at the terminal and update this value if necessary. 
+
+Multipass assigns a static IP address for you when you create a virtual machine.  It can be found by running `multipass list`
+from a MacOS terminal under the IPv4 column.  For convenience, adding a line to /etc/hosts will enable referring to this
+VM by its name rather than its IP address:
+_builder-vm   192.168.1.10_
+
+### VMWare Fusion
```
[the remark this answers is not in the document]
Comment (pozsgaic) [reply DEVELOPING.md:45]: Will remove VMWare specific content.

**Review by sajjaphani — APPROVED** [review]
Looking good!

**Pull request merged**